timeless/roles/models.py

```diff
@@ -1,6 +1,16 @@
 """File for models in roles module"""
 
 from timeless import DB
+from enum import Enum
+from sqlalchemy_utils.types.choice import ChoiceType
+
+
+class RoleType(Enum):
+    """ Types of roles """
+    Director = 1
```

Comment:
@ammaratef45 I don't like that much enums where keys are integers, please make them strings like:

`Director = "director"`

and so on